notebooks/community/pipelines/google_cloud_pipeline_template.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>Move H1 title to above links cell. Should precede the links.</p><p> </p><p>This section should start with Overview heading.</p><p> </p><p>Split into 4 text cells (Overview, Dataset, Objective, Costs).</p><p> </p><p>Move Dataset after Objective.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>